### ISSUE TITLE
Failed test debug -p

### ISSUE THREAD

**Issue by jacob-lee** [NYUCCL/psiTurk#451]
I resolved my issues getting pytest to run on my dev machine. When I did run it, I found that it was failing one test in test_do_commands in `test_psiturk_shell with command` `['debug -p']`

```
>           assert not captured.err
E           AssertionError: assert not 'generating debug url using `ad_url` config var\n'
E            +  where 'generating debug url using `ad_url` config var\n' = CaptureResult(out='[\x01\x02psiTurk\x01\x02 server:\x01\x02off\x01\x02 mode:\x01\x02sdbx\x01\x02 #HITs:0]$ mode sandbo...bugKTTJBW&hitId=debugDPJK04&workerId=debugIIT1PF&mode=debug\n', err='generating debug url using `ad_url` config var\n').err
```

The issue is that calls to Cmd2.pfeedback in `psiturk_shell.py` were added recently:

```python
try:
    base_url = self.config.get_ad_url()
    self.pfeedback('generating debug url using `ad_url` config var')
except PsiturkException:
    self.pfeedback('`ad_url_*` config vars not set; using Server '
                              'Parameters host and port vars.')
    host = self.config.get('Server Parameters', 'host')
    port = self.config.get('Server Parameters', 'port')
    base_url = f"http://{host}:{port}/ad"
```

Because `get_shell` in `test_psiturk_shell` creates a shell where the `quiet` parameter is set to False.

So the options are: 
1. ignore it, 
2. change the test,
3. change Cmd2 to output non-essential feedback to stdout instead of stderr.
4. remove call to pfeedback

I think #3 is best option. I've implemented in a PR request that I'll post momentarily.

**Comment by deargle** [comment]
Ah yeah, I wrote that test and then got distracted. But that test would have caught the earlier `debug -p` bug you found

**Comment by deargle** [comment]
Closed by #452
